src/converter/defaultConverter.ts

```diff
@@ -14,7 +15,14 @@ export default (
     date = parseISO(date);
   }
 
-  const { includeSeconds, addSuffix = true } = converterOptions;
+  const { includeSeconds, addSuffix = true, useStrict = false } = converterOptions;
+
+  if(useStrict) {
```

Comment:
Probably the reason why the pipeline (linting) fails
```suggestion
  if (useStrict) {
```